iidrees/More-Recipes
# Ch ci testing 152677333

### What does this PR do
Integration testing. This PR is expected to help test the API and its integration with the Postgres database it uses.

#### Description of Task to be completed
Clean up code and make it more robust

#### How should this be manually tested
* clone this repo
* at the root of this project directory run `npm install` 
* after installation run `npm test` to see the tests pass and coverage report.
#### What are the relevant pivotal tracker stories
#152677333
#### Screenshots
N/A
